### PR TITLE
Conclusion

### PR DESCRIPTION
**Pull request by EvanZhouDev**
Should be ready to merge... check it out, I guess.

### CODE REVIEW

**Comment by ThatXliner** [comment]
Suggestions:
 - Animations on icons to match what we did in intro
- The stylizing on the bullet list a bit sparse and weird. I can't quite put my finger on it yet. Maybe you need to add icons in front
- (are the following points on the rubric? I can't check right now)
 - On the "What we learned about the Holy Spirit"
   - "God promises the Holy Spirit as our Savior" should be changed to "God promises the Holy Spirit for guiding us" or something of the sort. *Jesus* is the savior, the Holy Spirit is the teacher.
   - The third point should be changed to "We can count on the Holy Spirit to stay with us"
 - On the "What we learned about the Christian Church" 
   - Talk more about Paul and the history of the church, replacing the second point.
   - Change "Believing in Jesus will save us" to "Accepting Jesus as your savior and believing that he died for your sins will save you into salvation"
 - Remove "The End."
 - Change copyright notice to "Copyright © 2022 Amaar Chughtai, Bryan Hu, and Yifan Zhou under the AGPL-3.0" because it looks better

**Review by ThatXliner — COMMENTED** [review]
Suggestions:

TL;DR: Change stylizing, change points because the points are cliche

- [x] Animations on icons to match what we did in intro

- [ ] The stylizing on the bullet list a bit sparse and weird. I can't quite put my finger on it yet. Maybe you need to add icons in front
- [ ] (are the following points on the rubric? I can't check right now)
 - [ ] On the "What we learned about the Holy Spirit"
   - [ ] "God promises the Holy Spirit as our Savior" should be changed to "God promises the Holy Spirit for guiding us" or something of the sort. *Jesus* is the savior, the Holy Spirit is the teacher.
   - [ ] The third point should be changed to "We can count on the Holy Spirit to stay with us"
 - [x] On the "What we learned about the Christian Church" 
   - [x] Talk more about Paul and the history of the church, replacing the second point.
   - [ ] Change "Believing in Jesus will save us" to "Accepting Jesus as your savior and believing that he died for your sins will save you into salvation"
 - [x] Remove "The End."
 - [x] Change copyright notice to "Copyright © 2022 Amaar Chughtai, Bryan Hu, and Yifan Zhou under the AGPL-3.0" because it looks better

**Comment by EvanZhouDev** [comment]
Fixes have been pushed!

**Review by ThatXliner — CHANGES_REQUESTED** [review]
LGTM :shipit:

**Comment by ThatXliner** [comment]
Oops

**Comment by ThatXliner** [comment]
Wait I forgot to undo the dev mode
